Python/import.c

```diff
@@ -2964,6 +2978,32 @@ PyImport_AppendInittab(const char *name, PyObject* (*initfunc)(void))
     return PyImport_ExtendInittab(newtab);
 }
 
+
+static bool
+_PyImport_LazyImportsIsEnabled()
+{
+    PyInterpreterState *interp = _PyInterpreterState_GET();
+    if (interp->lazy_imports_enabled ||
+        _PyInterpreterState_GetConfig(interp)->lazy_imports)
+    {
+        return true;
+    }
+    return false;
+}
+
+int
+PyImport_EnableLazyImports()
+{
+    PyInterpreterState *interp = _PyInterpreterState_GET();
+    interp->lazy_imports_enabled = 1;
+
+    if (_PyImport_LazyImportsIsEnabled()) {
```

Comment:
this check is also redundant, and this function should also be `void`
the only failure mode is if `interp` is NULL, and this would crash the process either way. maybe you can add an `assert(interp != NULL)`